project/plugins.sbt

```diff
@@ -12,3 +12,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.12.0")
 // need to add jdeb dependency explicitly because https://github.com/sbt/sbt-native-packager/issues/1053
 libraryDependencies += "org.vafer" % "jdeb" % "1.3" artifacts (Artifact("jdeb", "jar", "jar"))
 
+/*
+   See https://github.com/guardian/maintaining-scala-projects/issues/13
+ */
+libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
```

Comment:
See:

* https://github.com/guardian/maintaining-scala-projects/issues/13

In the future, this line of `libraryDependencySchemes` config will eventually be able to be removed, probably with:

* https://github.com/guardian/maintaining-scala-projects/issues/4